.github/workflows/update-subtree.yml

```
@@ -59,7 +59,24 @@ jobs:
         echo "CURRENT_COMMIT_HASH=${CURRENT_COMMIT_HASH}" >> $GITHUB_ENV
         echo "NEXT_COMMIT_HASH=${NEXT_COMMIT_HASH}" >> $GITHUB_ENV
```

Re: setting the Kani version too:

```suggestion
        echo "NEXT_COMMIT_HASH=${NEXT_COMMIT_HASH}" >> $GITHUB_ENV
        KANI_COMMIT_HASH=$(git -C kani-tmp rev-parse HEAD)
        echo "KANI_COMMIT_HASH=${KANI_COMMIT_HASH}" >> $GITHUB_ENV
```
I would not commit this suggestion verbatim, since Github won't let me make suggestions any higher in the step than this. Instead, I'd suggest 1) changing the title of this step to reflect that we're also grabbing the Kani toolchain version and 2) setting KANI_COMMIT_HASH up higher, along with the other variables, then echoing it to the environment down here.